src/Razor/Microsoft.AspNetCore.Mvc.Razor.Extensions/src/InjectDirective.cs

```diff
@@ -111,21 +111,5 @@ public override void VisitDirective(DirectiveIntermediateNode node)
                 }
             }
         }
-
-        #region Obsolete
-        [Obsolete("This method is obsolete and will be removed in a future version.")]
-        public static IRazorEngineBuilder Register(IRazorEngineBuilder builder)
```

Comment:
Should we keep these for 6.0? I thought our plan was to leave the `IRazorEngineBuilder` stuff in shipping code alone for 6.0 and post an announcement about removing this a future release.

Reply:
OK. I might've remembered incorrectly. I recall we had said we would keep the extension methods on IRazorEngineBuilder (since those were removed in the OG PR) but continue removing the implementations in the directives. Although now that I've typed this out, that assessment makes no sense. Will update accordingly.

Reply:
I should have been clearer. My suggestion was to leave everything around `IRazorEngineBuilder` in shipping APIs alone if we weren't doing removing them for this release.

Reply:
Makes sense. At this point, this PR is basically a vehicle for us to make the "hey this stuff is obsoleted" announcement before we move forward with the removal post-.NET 6.